srcpkgs/pantheon-session/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'pantheon-session'
+pkgname=pantheon-session
+version=20200331
+revision=1
+depends="gnome-session gala pipewire wingpanel pantheon-default-settings pantheon-dpms-helper pantheon-applications-menu"
```

Comment:
The applications menu should already pull in gala and wingpanel, no need to explicitly depend on them. Are you not missing plank though? And why is pipewire necessary?

Reply:
> plank-0.11.89_2 in transaction breaks installed pkg `gala-3.2.0_1'
gala pull in plank.